index.js

```diff
@@ -128,10 +129,34 @@ class TypeWriter extends Component {
   render() {
     const { children, ...props } = this.props;
     const { visibleChars } = this.state;
+    const { fixed } = this.props;
```

Comment:
This could be added to the destructuring assignment above with `children` and `...props`.